modules/yieldlabBidAdapter.js

```diff
@@ -83,6 +80,29 @@ export const spec = {
       if (floor) {
         adslotFloors.push(bid.params.adslotId + ':' + floor);
       }
+      const dsaRequired = bid.params.regs?.ext?.dsa?.required;
```

Comment:
you need to follow the spec; please look at the appnexus and magnite pr's for the location of dsa information in the bid request object.

Reply:
Thank you, I updated the pr.